_posts/2019-02-28-into-the-darkness-of-viewmodel-lists-composition.md

```diff
@@ -13,15 +13,17 @@ tags:
 
 We had [a first look at what problem ViewModel Composition is](https://milestone.topics.it/2019/02/06/what-is-services-viewmodel-composition-again.html) designed to solve, then we analyzed what it means to [compose a single item](https://milestone.topics.it/2019/02/20/viewmodel-composition-maze.html), using a Product as sample model. It's now time to start exploring what it means to compose a list of Product ViewModels.
 
-## Dark times
```

Comment:
I'm not happy with my replacement title but I thought something more descriptive would be useful.

Reply:
I agree with the descriptiveness, the original idea (Dark times) was to find a correlation with the post title: Into the darkness of ViewModels Lists composition.

Reply:
Accepting this as is, it’s better than mine. I’ll keep a thread running in my head to get back to this sometime in the future.